libpicotcp/CMakeLists.txt

```diff
@@ -72,6 +72,11 @@ if(LibPicotcp)
     add_custom_target(get_muslc)
     add_dependencies(get_muslc muslc)
 
+    set(sed_inplace_suffix "")
```

Comment:
Need a comment explaining why the behaviour is needed on macOS.

Reply:
Yes, a comment here would be nice